tests/utils.ts

```diff
@@ -0,0 +1,46 @@
+import { test, expect } from '@playwright/test';
+import { get } from 'http';
+
+const api = (request) => {
```

Comment:
Where are we adding the authentication token?

Reply:
It's done in global-setup.ts  https://github.com/dev-launchers/strapiv4/pull/163/files#diff-a9f8e507c1cbc8c53302d037e01991ce78517f1ac3ca248134a6a1f42d86fd79R14 and https://github.com/dev-launchers/strapiv4/pull/163/files#diff-f679bf1e58e8dddfc6cff0fa37c8e755c8d2cfc9e6b5dc5520a5800beba59a92R22